pkg/storaged/stratis-details.jsx

```diff
@@ -499,25 +523,45 @@
                     CheckBoxes("mount_options", _("Mount options"),
                                {
                                    value: {
-                                       auto: true,
                                        ro: false,
-                                       never_auto: false,
                                        extra: false
                                    },
                                    fields: [
-                                       { title: _("Mount now"), tag: "auto" },
                                        { title: _("Mount read only"), tag: "ro" },
-                                       {
-                                           title: _("Never mount at boot"),
-                                           tag: "never_auto",
-                                           tooltip: never_auto_explanation,
-                                       },
                                        { title: _("Custom mount options"), tag: "extra", type: "checkboxWithInput" },
                                    ]
-                               })
+                               }),
+                    SelectOne("at_boot", _("At boot"),
+                              {
+                                  value: "nofail",
+                                  explanation: mount_explanation.nofail,
+                                  choices: [
+                                      {
+                                          value: "local",
+                                          title: _("Mount before services start"),
+                                      },
+                                      {
+                                          value: "nofail",
+                                          title: _("Mount without waiting, ignore failure"),
+                                      },
+                                      {
+                                          value: "netdev",
+                                          title: _("Mount after network becomes available, ignore failure"),
+                                      },
+                                      {
+                                          value: "never",
+                                          title: _("Do not mount"),
+                                      },
```

Comment:
These 22 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221123-133546-fbb094c1-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)